libs/ui-kit/toggletip/toggletip.component.scss

```diff
@@ -0,0 +1,50 @@
+cz-toggletip {
+  display: none;
+}
+
+.cz-toggletip-content {
+  --bg-color: var(--cz-color-white);
+  --font-size: 0.875rem;
+  position: relative;
+  max-width: 23rem;
```

Comment:
oui :D can't we have this in px? (since we want to use px for width at this moment 🤔 )

Reply:
Here it's one of the cases where width in rem makes more sense -> if the user sets a high font size, it's better to leave it some more breathing room, otherwise with a width in px it will feel cramped 😅

Reply:
ok, good to know :)